packages/ckeditor5-emoji/README.md

```diff
@@ -3,24 +3,30 @@ CKEditor&nbsp;5 emoji feature
 
 [![npm version](https://badge.fury.io/js/%40ckeditor%2Fckeditor5-emoji.svg)](https://www.npmjs.com/package/@ckeditor/ckeditor5-emoji)
 [![Coverage Status](https://coveralls.io/repos/github/ckeditor/ckeditor5/badge.svg?branch=master)](https://coveralls.io/github/ckeditor/ckeditor5?branch=master)
-[![Build Status](https://travis-ci.com/ckeditor/ckeditor5.svg?branch=master)](https://app.travis-ci.com/github/ckeditor/ckeditor5)
+[![CircleCI](https://circleci.com/gh/ckeditor/ckeditor5.svg?style=shield)](https://app.circleci.com/pipelines/github/ckeditor/ckeditor5?branch=master)
 
 This package implements the emoji feature for CKEditor&nbsp;5.
 
+## Installation
+
+This plugin is part of the `ckeditor5` package. Install the whole package to use it.
+
+```bash
+npm install ckeditor5
+```
+
+## Create free account
+
+If you want to check full CKEditor&nbsp;5 capabilities, sign up for a [free non-commitment 14-day trial](https://portal.ckeditor.com/checkout?plan=free).
+
 ## Demo
 
-Check out the demo in the [emoji feature guide](https://ckeditor.com/docs/ckeditor5/latest/features/emoji.html#demo).
+Check out the [demo in the emoji feature guide](https://ckeditor.com/docs/ckeditor5/latest/features/emoji.html#demo).
```

Comment:
This is totally not clear to me if we should also change this text to "demo in the Emoji feature" - with upper case feature name.

All other packages have mixed up feature names in multiple places: sometimes they are written with a capital letter and sometimes with a lower case letter, so probably we should keep it here as it is now. But, I'm not sure 🤔